source/guides/1.1/getting-started.md

```diff
@@ -506,8 +506,7 @@ Let's write a test to force this change in our view:
 
 ```ruby
 # spec/web/views/books/index_spec.rb
-require 'spec_helper'
-require_relative '../../../../apps/web/views/books/index'
```

Comment:
The generator also doesn't require the files. Again: I think either the generator or the guide should be adjusted.